### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by schoren**
This PR updates the README

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by jorgeepc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,17 +34,17 @@
 
 # Tracetest
 
-Tracetest is a trace-based testing tool that leverages the data contained in your distributed traces to produce easy to create, yet super powerful integration tests. You can verify activity deep inside your system by asserting on data and flow information contained in the OpenTelemetry traces and span attributes. This can include:
+Tracetest is a trace-based testing tool that leverages the data captured by your exiting Open Telemetry distributed traces to produce easy to create, yet super powerful integration tests. You can verify activity deep inside your system by asserting on data and flow information contained in the OpenTelemetry traces and span attributes. This can include:
 
+- verify the quality of your OpenTelemetry instrumentation and enforce standards.
```
Comment (jfermi) [inline README.md:39]:
```suggestion
- Verifying the quality of your OpenTelemetry instrumentation and enforcing standards.
```

File: README.md [hunk 2]
```diff
@@ -34,17 +34,17 @@
...
 - testing events that occur on 'the other side' of an async message queue, even though the original async call has returned earlier.
```
Comment (jfermi) [inline README.md:40]:
```suggestion
- Testing events that occur on 'the other side' of an async message queue, even though the original async call has returned earlier.
```

File: README.md [hunk 3]
```diff
@@ -34,17 +34,17 @@
...
 - assertions based on the timing of different steps in your process.
```
Comment (jfermi) [inline README.md:41]:
```suggestion
- Assertions based on the timing of different steps in your process.
```

File: README.md [hunk 4]
```diff
@@ -34,17 +34,17 @@
...
 - wildcard assertions across common types of activities, ie all gRPC return codes should be 0, all database calls should happen in less than 100ms.
```
Comment (jfermi) [inline README.md:42]:
```suggestion
- Wildcard assertions across common types of activities, ie all gRPC return codes should be 0, all database calls should happen in less than 100ms.
```

File: README.md [hunk 5]
```diff
@@ -34,17 +34,17 @@
...
 - test long running processes instrumented with OpenTelemetry tracing to assert proper operation deep in the process.
```
Comment (jfermi) [inline README.md:43]:
```suggestion
- Testing long running processes instrumented with OpenTelemetry tracing to assert proper operation deep in the process.
```

File: README.md [hunk 6]
```diff
@@ -34,17 +34,17 @@
...
-- verify the quality of your OpenTelemetry instrumentation and enforce standards.
 
 # Features
 
-- Test by executing a REST or gRPC call to trigger the test. Can also import your Postman Collections.
+- Supports multiple ways of creating a test, including HTTP, GRPC and Postman Collections.
```
Comment (jfermi) [inline README.md:47]:
```suggestion
- Supporting multiple ways of creating a test, including HTTP, GRPC and Postman Collections.
```

File: README.md [hunk 7]
```diff
@@ -34,17 +34,17 @@
...
 - [Add assertions](https://kubeshop.github.io/tracetest/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.
```
Comment (jfermi) [inline README.md:48]:
```suggestion
- [Adding assertions](https://kubeshop.github.io/tracetest/adding-assertions/) based on return data from trigger call and/or data contained in the spans in your distributed trace.
```

File: README.md [hunk 8]
```diff
@@ -34,17 +34,17 @@
...
 - Specify which spans to check in assertions via the [advanced selector language](https://kubeshop.github.io/tracetest/advanced-selectors/).
```
Comment (jfermi) [inline README.md:49]:
```suggestion
- Specifying which spans to check in assertions via the [advanced selector language](https://kubeshop.github.io/tracetest/advanced-selectors/).
```

File: README.md [hunk 9]
```diff
@@ -34,17 +34,17 @@
...
 - Define checks against the attributes in these spans, including properties, return status, or timing.
```
Comment (jfermi) [inline README.md:50]:
```suggestion
- Defining checks against the attributes in these spans, including properties, return status, or timing.
```

File: README.md [hunk 10]
```diff
@@ -117,7 +116,7 @@ testDefinition:
 
 # Feedback
 
-We are in the our early days with the project and need your help. Have an idea to improve it? Please [Create an issue here](https://github.com/kubeshop/tracetest/issues/new/choose) or join our community on [Discord](https://discord.gg/eBvEQRVyKX).
+Have an idea to improve Tracetest? Please [Create an issue here](https://github.com/kubeshop/tracetest/issues/new/choose) or join our community on [Discord](https://discord.gg/eBvEQRVyKX).
```
Comment (jfermi) [inline README.md:119]:
```suggestion
Have an idea to improve Tracetest? Please [create an issue here](https://github.com/kubeshop/tracetest/issues/new/choose) or join our community on [Discord](https://discord.gg/eBvEQRVyKX).
```

File: README.md [hunk 11]
```diff
@@ -117,7 +116,7 @@ testDefinition:
...
 
 Follow us on [Twitter at @tracetest_io](https://twitter.com/tracetest_io) for updates
```
Comment (jfermi) [inline README.md:121]:
```suggestion
Follow us on [Twitter at @tracetest_io](https://twitter.com/tracetest_io) for updates.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Review by kdhamric — APPROVED** [review]

**Review by xoscar — APPROVED** [review]

**Pull request merged**